pkg/controller/build/buildrequest/assets/buildah-build.sh

```diff
@@ -5,11 +5,32 @@
 # custom build pod.
 set -xeuo
 
+DEST=/etc/pki/ca-trust/extracted
+
+# Prevent p11-kit from reading user configuration files.
+export P11_KIT_NO_USER_CONFIG=1
+
+# OpenSSL PEM bundle that includes trust flags
+/usr/bin/p11-kit extract --format=openssl-bundle --filter=certificates --overwrite --comment $DEST/openssl/ca-bundle.trust.crt
+/usr/bin/p11-kit extract --format=pem-bundle --filter=ca-anchors --overwrite --comment --purpose server-auth $DEST/pem/tls-ca-bundle.pem
+
+su -m build << 'EOF'
+set -xeuo
```

Comment:
thought (non-blocking): In the future, it would be great if we could migrate this to a separate script. Then we could do `su -m build ./separate-script.sh`.

To be clear: I'm not asking you to do that now because we presently don't have a great way to do that. When we consider using a different builder entrypoint instead of this script, we can address it then.